test/e2e/crd_e2e_test.go

```diff
@@ -516,3 +519,19 @@ var _ = Describe("CRD Versions", func() {
 		GinkgoT().Log("manually reconciled potentially unsafe CRD upgrade")
 	})
 })
+
+func fetchCRD(c operatorclient.ClientInterface, crc versioned.Interface, name, namespace string) (*v1beta1.CustomResourceDefinition, error) {
```

Comment:
namespace argument is not used here since a CRD is cluster scoped

Reply:
removed, namespace arg does not make sense here.